pom.xml

```diff
@@ -129,6 +129,13 @@
       <version>1.19.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <!-- to manipulate DNS servers for testing -->
```

Comment:
:cold_sweat:

Reply:
better than telling people to add "server 127.0.0.1" to their resolve.conf (and disabling the systemd-resolver) just to run a test.

there was a way prior to java9 where you could set a system property and that would then work but it was removed without a reasonable replacement (you can now supply something like an `/etc/hosts` file but it will error if a host is not found in it, and it can not be used for SRV lookups